test/libyul/yulOptimizerTests/fakeStackLimitEvader/stub.yul

```diff
@@ -0,0 +1,88 @@
+{
+	mstore(0x40, memoryguard(0))
+	function f() {
+	    let $fx
+	    let $fy := 42
+	    sstore($fx, $fy)
+	    $fx := 21
+	}
+	function g(gx) {
+	    let $gx, $gy := tuple2()
+	    { $gx, $gy := tuple2() }
+	    { $gx, gx := tuple2() }
+	    { gx, $gy := tuple2() }
+	}
+	function h(hx, hy, hz, hw) {
+	    let $hx, $hy, $hz, $hw := tuple4()
+	    { hx, $hy, hz, $hw := tuple4() }
+	    { $hx, $hy, hz, hw := tuple4() }
+	}
+	function tuple2() -> a, b {}
+	function tuple4() -> a, b, c, d {}
+    f()
+    g(0)
+    h(1, 2, 3, 4)
+}
+// ----
+// step: fakeStackLimitEvader
+//
+// {
+//     mstore(0x40, memoryguard(0x80))
+//     function f()
+//     {
+//         mstore(0x20, 0)
+//         mstore(0x00, 42)
+//         sstore(mload(0x20), mload(0x00))
+//         mstore(0x20, 21)
+//     }
+//     function g(gx)
+//     {
+//         let $gx_1, $gy_2 := tuple2()
+//         mstore(0x20, $gy_2)
```

Comment:
So in this test `0x20` can be used by both `f` and `g` because they're not in the same call path, right?

Reply:
Exactly.